### PR TITLE
updated docs for slack alerts webhooks

### PR DESCRIPTION
**Pull request by TehreemNisa**
updated docs for slack alerts webhooks and slack apps update

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-223-f3b4eb93`

File: content/sre/alerting/log-alerts.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ To configure log alerting do following:
 
 ## 1. Configure the incoming webhook in slack
 
+> Note : use slack bot account to manage incoming webhooks
```
Comment (usamaahmadkhan) [inline content/sre/alerting/log-alerts.md]: Always use Slack bot account to manage incoming webhooks. An integration/app might fail if the user that added it leaves

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: content/sre/alerting/log-alerts.md (hunk 1 above)
Comment (TehreemNisa) [reply content/sre/alerting/log-alerts.md]: updated

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-223-17e3891b`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.162`

**Pull request merged**